tests/multi_prover.rs

```diff
@@ -83,6 +83,22 @@ async fn multi_prover() -> anyhow::Result<()> {
     let uri = "http://".to_owned() + &local_addr.to_string();
     let client = Client::new();
 
+    // Insert provers to database
+    test_add_prover(
+        &uri,
+        &client,
+        &prover_map[&batch_size_3],
+        batch_timeout_seconds as usize,
+    )
+    .await?;
+    test_add_prover(
+        &uri,
+        &client,
+        &prover_map[&batch_size_10],
+        batch_timeout_seconds as usize,
+    )
+    .await?;
```

Comment:
Could you create a function that accepts a list (mapping?) of provers and adds all of them to given sequencer?

Actually since we're mocking provers, we could probably extract their creation from `spawn_deps` and move it after the sequencer is up and running